[Inductor] address comments from https://github.com/pytorch/pytorch/pull/163803

hl475 opened the pull request:
Summary: address comments from https://github.com/pytorch/pytorch/pull/163803

Differential Revision: D83291637




cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @chenyang78 @kadeng @muchulee8 @amjames @chauhang @aakhundov @coconutruben

hl475 commented:
@pytorchbot label "topic: not user facing"

desertfire commented on torch/_inductor/decomposition.py:
```diff
@@ -885,11 +885,13 @@ def select_decomp_table() -> dict[Any, Callable[..., Any]]:
         for k in decompositions.keys():
             if k == torch.ops.quantized.embedding_bag_byte_unpack.default:
-                del decompositions[k]
+                decompositions.pop(
```
I think you can get rid of the for-loop by using pop.

hl475 replied:
you are right!

hl475 committed:
[Inductor] address comments from https://github.com/pytorch/pytorch/pull/163803 (#163901)

Summary:

address comments from https://github.com/pytorch/pytorch/pull/163803

Differential Revision: D83291637
